src/_common/content/content-editor/modals/gif/gif-modal.styl

```diff
@@ -157,6 +159,18 @@ $result-border-width = 4px
 		border-color: var(--theme-backlight)
 		background-color: var(--theme-backlight)
 
+	&-favorite-icon
```

Comment:
don't use &. Just put `.search-result-favorite-icon`. we switched to not using & syntax to make it easier to search for used classes

Reply:
You can use it in :hover of course, of if you need to do something like `&.extra-class`. It's just that we want the classes fully spelled out for easy searching